content/client-lib-development-guide/features.textile

```diff
@@ -549,6 +559,17 @@ h4. TokenParams
 * @(TK2c)@ @clientId@ string -  A @clientId@ to associate with this token
 * @(TK2d)@ @timestamp@ long - The timestamp (in millis since the epoch) of this request. Timestamps, in conjunction with the @nonce@, are used to prevent n requests from being replayed
 
+h4. AuthOptions
+* @(AO1)@ A class providing configurable authentication options used when authenticating or issuing tokens explicitly. These options are used when invoking @Auth#authorise@, @Auth#requestToken@ and @Auth#createTokenRequest@
+* @(AO2)@ The attributes of @AuthOptions@ consist of:
+** @(AO2a)@ @key@ string - Full Ably key string, as obtained from dashboard, used for all token requests
```

Comment:
"Used when signing token requests locally". Otherwise it sounds like you need it to perform any token request, which isn't true.